gapis/service/service.proto

```diff
@@ -397,6 +397,9 @@ message ErrDataUnavailable {
   stringtable.Msg reason = 1;
   // If true, then making the same request at a later time may result in data.
   bool transient = 2;
+  // If true, draw call specific data was requested for a command range not
+  // containing any draw calls.
+  bool not_a_draw_call = 3;
```

Comment:
> If you really want to do this, then I suggest we make the stringtable message types (`ERR_NOT_A_DRAW_CALL`) 

After more consideration, this is also a bad idea. Detailed error messages should also support additional data fields which stringtables were not designed to carry (in any sensibly structured form). If we really need to convey additional error details, then they should go into a new `ErrXXX` message type, but I don't think that's the case here.

Generally, I'm thinking attempting to redeclare the presented string the client is a bad idea.

Who's to say draw calls are the only thing that gets a geometry view? The API makes no such constraint:
```proto
// Mesh is a path to a mesh representation of an object.
message Mesh {
    MeshOptions options = 1;
    oneof object {
        Command command = 2;
        CommandTreeNode command_tree_node = 3;
    }
}
```

We may decide in Vulkan to display the only draw call in a queue submit (as we do now), or decide that you have to actually select that subcommand.
We may decide that we could show the geometry on a `glBufferData` command, if we're clever enough to determine that this is geometry data. etc.
Any one of these methods of finding a mesh could fail in exciting ways that we may want to change the error message for. "Mesh has no position data" is just one example that springs to mind. Does that get the special no-icon treatment? If not, why?

Going back to your original goal:

> In this circumstance, I want to have three different states: 1) success, 2) not a draw call, 3) error. 2) **shows a specific message without an error icon and feel**: "Select a draw call". From the client's perspective, the no draw call error **is expected, but unavoidable**, that should be treated differently than any other error.

Let's start with the "**is expected, but unavoidable**" bit. That's what `ErrDataUnavailable` is supposed to represent. It's a "soft" error, in that it should be entirely expected by the client and the `Reason` can optionally be presented to the user.

"shows a specific message without an error icon and feel" - maybe we could add something to the error message to describe the severity, or some other hints that gives you what you need here - but it sounds like you're describing exactly how I intended `ErrDataUnavailable` to be presented. Are there cases where you're getting a `ErrDataUnavailable` where you feel it should be displayed as more of a critical error?

I don't want to appear to be over-reacting to this, but I'm concerned about making apparently small changes to the service which may have huge consequences down the line.